### DISCUSSION TITLE
socket error: EOF

### DISCUSSION THREAD

**Discussion by hebe061103** [soranoo/TradingView-Free-Webhook-Alerts#67] [General]
socket error: EOF 
这是什么问题？我使用的是邮件接收。

**Comment by soranoo** [comment]
May I have the log?

**Reply by hebe061103** [reply]
│ /usr/local/lib/python3.10/dist-packages/imapclient/imapclient.py:1627 in                         │
│ _consume_until_tagged_response                                                                   │
│                                                                                                  │
│   1624 │   │   tagged_commands = self._imap.tagged_commands                                      │
│   1625 │   │   resps = []                                                                        │
│   1626 │   │   while True:                                                                       │
│ ❱ 1627 │   │   │   line = self._imap._get_response()                                             │
│   1628 │   │   │   if tagged_commands[tag]:                                                      │
│   1629 │   │   │   │   break                                                                     │
│   1630 │   │   │   resps.append(_parse_untagged_response(line))                                  │
│                                                                                                  │
│ /usr/lib/python3.10/imaplib.py:1075 in _get_response                                             │
│                                                                                                  │
│   1072 │   │   # Returns None for continuation responses,                                        │
│   1073 │   │   # otherwise first response line received.                                         │
│   1074 │   │                                                                                     │
│ ❱ 1075 │   │   resp = self._get_line()                                                           │
│   1076 │   │                                                                                     │
│   1077 │   │   # Command completion response?                                                    │
│   1078                                                                                           │
│                                                                                                  │
│ /usr/lib/python3.10/imaplib.py:1185 in _get_line                                                 │
│                                                                                                  │
│   1182 │   │                                                                                     │
│   1183 │   │   line = self.readline()                                                            │
│   1184 │   │   if not line:                                                                      │
│ ❱ 1185 │   │   │   raise self.abort('socket error: EOF')                                         │
│   1186 │   │                                                                                     │
│   1187 │   │   # Protocol mandates all lines terminated by CRLF                                  │
│   1188 │   │   if not line.endswith(b'\r\n'):                                                    │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯

**Comment by soranoo** [comment]
It means the connection failed, it may caused by timeout.

**Reply by hebe061103** [reply]
现在使用你的云端版,感觉比email快
